library/std/src/sys/unix/ext/fs.rs

```diff
@@ -345,6 +345,33 @@ pub trait OpenOptionsExt {
     /// ```
     #[stable(feature = "open_options_ext", since = "1.10.0")]
     fn custom_flags(&mut self, flags: i32) -> &mut Self;
+
+    /// Get the flags of this OpenOptions as libc::c_int.
```

Comment:
Can we link `libc::c_int`?

Reply:
Is it enough to put it in brackets like this?
 
    [`libc::c_int`]

Reply:
Solved using [this](https://github.com/rust-lang/api-guidelines/issues/186) as a reference.